File: src/Atividade30.java
import java.util.Scanner;

public class Atividade30 {
    public static void main(String[] args) {
        var entrada = new Scanner(System.in);
        boolean continuar = true;

        while (continuar){
            System.out.println("-------------------------");
            System.out.println("------Atividade 30-------");
            System.out.println("-------------------------");

            System.out.println("Digite um numero: ");
            var n1 = Integer.parseInt(entrada.nextLine());

            System.out.println("Digite um segundo numero: ");
            var n2 = Integer.parseInt(entrada.nextLine());

            System.out.println("Digite um terceiro numero: ");
            var n3 = Integer.parseInt(entrada.nextLine());

            int maiornum = Math.max(n1, Math.max(n2 ,n3));

            System.out.println("O maior numero é "+maiornum);

            System.out.println("Deseja saber qual o numero é maior novamente? ");
            var resposta = entrada.nextLine();

            if(resposta.equals("Não") || resposta.equals("NÃO") || resposta.equals("não") || resposta.equals("nao")){
                System.out.println("Saindo do Sistema......");
                continuar = false;
            }
        }
    }
}

File: src/Atividade30.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int n1,n2,n3;

    printf("\n================================="); 
    printf("\n==========Atividade 30===========");
    printf("\n================================="); 

    printf("Digite um numero: ");
    scanf("%d", &n1);

    printf("Digite um segundo numero: ");
    scanf("%d", &n2);

    printf("Digite um terceiro numero: ");
    scanf("%d", &n3);

    if (n1 > n2 || n1 > n3){
        printf("%d e o maior numero.", n1);
    }else if (n2 > n1 || n2 > n3){
        printf("%d e o maior numero.", n2);
    }else{
        printf("%d e o maior numero.", n3);
    }
    
}